blocks/results-list-block/features/results-list/results/result-item.jsx

```diff
@@ -53,9 +55,16 @@ const ResultItem = React.memo(React.forwardRef(({
           </div>
         )
         : null }
-      {(showHeadline)
+      {(showHeadline || showItemOverline)
```

Comment:
This logic is causing an issue with toggling the headline, this will pass if either showHeadline or showItemOverline is true and a few lines below the headline is always shown if this evaluates to true

Reply:
fixed